packages/klevu-core/src/connection/klevuFetch.ts

```diff
@@ -127,6 +129,38 @@ function KlevuQueriesById(
   }
 }
 
+/**
```

Comment:
This new code could moved in it's own file so that KlevuFetch.ts doesn't clutter up